node/tessel-export.js

```diff
@@ -557,17 +562,21 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
       throw new Error(`Interrupts are not supported on pin ${this.pin}. Pins 2, 5, 6, and 7 on either port support interrupts.`);
     }
 
-    if ((mode === 'high' || mode === 'low') && !callback.listener) {
-      throw new Error('Cannot use "on" with level interrupts. You can only use "once".');
-    }
-
-    if (this.interruptMode !== mode) {
+    // For one-time 'low' or 'high' event
+    if (mode === 'low' || mode === 'high') {
+      if (!callback.listener) {
+        throw new Error('Cannot use "on" with level interrupts. You can only use "once".');
+      }
+      // Can't set multiple listeners when using 'low' or 'high'
```

Comment:
I made the mistake of assuming that the `.addListener()` function was called for all added listeners, but I see that it is only called for `.on()` listeners, not `.once()` listeners.

Where/how does the `pin.interruptMode` get set when doing a `pin.once('high')` or `pin.once('low')`?